en/docs/install-and-setup/perfromance-tuning-and-test-results/tuning-performance.md

```diff
@@ -32,81 +32,83 @@ Following are the configurations that can be applied to optimize the OS-level pe
 1.  To optimize network and OS performance, configure the following settings in the `/etc/sysctl.conf` file of Linux. These settings specify a larger port range, a more effective TCP connection timeout value, and a number of other important parameters at the OS-level.
 
     !!! info
-            It is not recommended to use `net.ipv4.tcp_tw_recycle = 1` when working with network address translation (NAT), such as if you are deploying products in EC2 or any other environment configured with NAT.
+        It is not recommended to use `net.ipv4.tcp_tw_recycle = 1` when working with network address translation (NAT), such as if you are deploying products in EC2 or any other environment configured with NAT.
 
 
     ``` java
-     net.ipv4.tcp_fin_timeout = 30
-     fs.file-max = 2097152
-     net.ipv4.tcp_tw_recycle = 1
-     net.ipv4.tcp_tw_reuse = 1
-     net.core.rmem_default = 524288
-     net.core.wmem_default = 524288
-     net.core.rmem_max = 67108864
-     net.core.wmem_max = 67108864
-     net.ipv4.tcp_rmem = 4096 87380 16777216
-     net.ipv4.tcp_wmem = 4096 65536 16777216
-     net.ipv4.ip_local_port_range = 1024 65535      
+    net.ipv4.tcp_fin_timeout = 30
+    fs.file-max = 2097152
+    net.ipv4.tcp_tw_recycle = 1
+    net.ipv4.tcp_tw_reuse = 1
+    net.core.rmem_default = 524288
+    net.core.wmem_default = 524288
+    net.core.rmem_max = 67108864
+    net.core.wmem_max = 67108864
+    net.ipv4.tcp_rmem = 4096 87380 16777216
+    net.ipv4.tcp_wmem = 4096 65536 16777216
+    net.ipv4.ip_local_port_range = 1024 65535      
     ```
```

Comment:
Have we explained or given some reference to what these configurations mean?

Reply:
@chamilaadhi , please provide a reference to this configuration.

Reply:
https://wiki.archlinux.org/index.php/Sysctl

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/798/commits/ad35368762f90e93da8563604b42628e902ea243